tests/variable/variable.cpp

```diff
@@ -429,6 +429,30 @@ TEST(Variable, setGetInterfaceType)
     EXPECT_EQ(interfaceTypeString4, v4->interfaceType());
 }
 
+TEST(Variable, hasInterfaceType)
```

Comment:
Might want to test against the `none`, `public`, `private` and `public_and_private` interfaces after each call to `setInterfaceType()`/`removeInterfaceType()`?

Reply:
I think making sure the previous is gone and the new is in is sufficient here.